docs/serving/samples/knative-routing-go/README.md

```diff
@@ -67,14 +83,15 @@ docker push "${REPO}/docs/serving/samples/knative-routing-go"
    - Manually replace:
      `image: github.com/knative/docs/docs/serving/samples/knative-routing-go`
      with
-     `image: <YOUR_CONTAINER_REGISTRY>docs/serving/samples/knative-routing-go`
+     `image: ${REPO}/knative-routing-go`
+     You have to replace ${REPO} with the actual value on your local machine, if you manually change this yaml file.
```

Comment:
this sentence here is confusing

Reply:
If the value of ${REPO} is docker.io/myname, we nee to replace with  docker.io/myname/knative-routing-go, not ${REPO}/knative-routing-go.
This is what I meant.

Reply:
Suggestion, @houshengbo  : "If you manually changed the `.yaml` file, you must replace `${REPO}` with the correct path on your local machine."

Reply:
Done, thanks.